### PR TITLE
v1.3 Fix Sequence Diagrams

### LINKED ISSUE #134: Fix Delete Command Sequence Diagram.
The current Delete Command Sequence Diagram should indicate the end of the lifeline for the `DeleteCommand`

image

### PR DESCRIPTION
**Pull request by guohuang88**
This PR will
- Update DeleteSequenceDiagram to indicate end of lifeline for DeleteCommand
- Update FindSequenceDiagram to indicate end of lifeline for FindCommand
- Update FindTagsSequenceDiagram to indicate end of lifeline for FindTagsCommand

Fixes #134 #135 #140

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -657,7 +657,7 @@ Step 4: The user executes the `delete 1` command to remove the first patient in
 
 <box type="info" seamless>
 
-**Note:** The lifeline for `DeleteCommandParser` should end at the destroy marker (X) but
+**Note:** The lifeline for `DeleteCommandParser` and `DeleteCommand` should end at the destroy marker (X) but
```
Comment (ZhiWei1010) [inline docs/DeveloperGuide.md:660]: Good catch!

**Review by choowengyan — APPROVED** [review]
Well done on the fixes! LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -761,8 +761,8 @@ The following UML sequence diagram illustrates how the FindTags operations works
 
 <box type="info" seamless>
 
-**Note:** The lifeline for `FindTagsCommandParser` should end at the destroy marker (X) but
-due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+**Note:** The lifeline for `FindTagsCommandParser` and `FindTagsCommand` should end at the destroy marker
```
Comment (ZhiWei1010) [inline docs/DeveloperGuide.md:764]: Good catch too!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -715,7 +715,7 @@ The following UML sequence diagram illustrates how the Find operations works.
 
 <box type="info" seamless>
 
-**Note:** The lifeline for `FindCommandParser` should end at the destroy marker (X) but
+**Note:** The lifeline for `FindCommandParser` and `FindCommand` should end at the destroy marker (X) but
```
Comment (ZhiWei1010) [inline docs/DeveloperGuide.md:718]: Good catch!

**Review by ZhiWei1010 — APPROVED** [review]
LGTM!

**Pull request merged**